File: JAVA/04_While/BOJ_03_1110_더하기사이클.java
import java.io.IOException;
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        int m = n, cnt = 0;
        while (true) {
            m = (((m % 10) * 10) + (m / 10 + m % 10) % 10);
            cnt++;
            if (m == n) break;
        }
        System.out.print(cnt);
    }
}

File: C/04_While/BOJ_03_1110_더하기사이클.c
#include <stdio.h>
main() {
	int n, m, cnt;
	cnt = 1;
	scanf("%d", &n);
	if (n < 10) {
		n = n * 10;
	}
	m = n;
	while (((n % 10) * 10) + (((n / 10) + (n % 10)) % 10) != m) {
		n = ((n % 10) * 10) + (((n / 10) + (n % 10)) % 10);
		cnt++;
	}
	printf("%d", cnt);
}